source/Ship.cpp

```diff
@@ -3174,7 +3175,7 @@ int Ship::JumpsRemaining(bool followParent) const
 	{
 		// If this ship has no destination, the parent's substitutes for it,
 		// but only if the location is reachable.
-		auto p = GetParent();
+		const auto &p = GetParent();
```

Comment:
GetParent returns a shared pointer. Looking to other instances of GetParent in this file, none of them assign the result to a const reference like this, so think change and the same line in NeedsFuel seem incorrect to me.